stubs/braintree/braintree/apple_pay_card.pyi

```diff
@@ -1,12 +1,12 @@
-from typing import Any
+from typing import Any, Final
 
 from braintree.resource import Resource as Resource
 
 class ApplePayCard(Resource):
     class CardType:
-        AmEx: str
-        MasterCard: str
-        Visa: str
+        AmEx: Final = "Apple Pay - American Express"
+        MasterCard: Final = "Apple Pay - MasterCard"
+        Visa: Final = "Apple Pay - Visa"
```

Comment:
Personal opinion: These user facing strings with no further meaning to the computer should just be `Final[str] = ...`, while something like `GatewayRejected: Final = "gateway_rejected"` below makes sense.

Reply:
Also just a personal opinion: I feel it's easier to indiscriminately copy&paste from source, w/o having to think too much about what arbitrarily counts as an identifier vs a human readable string, which other contributors may not notice and follow in the same way.

There's an argument about the risk of the string changing, causing more churn, but I doubt these are likely to change without a major revision:
- braintree_python won't even remove dead code without a major version bump
- this card type, for example, is part of the result object for a transaction, it's likely users already highly depend on the value of this string for their transaction logs